src/pages/venues/index.js

```diff
@@ -27,12 +28,14 @@ export default function VenueListPage() {
           <PlaceBig />
         </StyledIconLink>
       </StyledHeader>
-      <VenueFilter />
-      {data?._embedded ? (
-        <VenueList venues={data._embedded.venues} />
-      ) : (
-        <p>No venues found. Adjust filter.</p>
-      )}
+      <StyledContent>
+        <VenueFilter />
+        {data?._embedded ? (
+          <VenueList venues={data._embedded.venues} />
+        ) : (
+          <p>No venues found. Adjust filter.</p>
```

Comment:
"Please adjust filter" stay polite to your user :joy:

Reply:
Ok if I really have to. They should be thankful though xD